deployment/packet/README.md

```diff
@@ -155,13 +165,13 @@ $ export PATH=/usr/local/go/bin:$PATH
 and follow the instructions on the [metrics VMs](https://github.com/kata-containers/ci/tree/master/VMs/metrics#example)
 page to complete the installation ready to deploy the instance into the CI Jenkins master.
 
-## Configuring checkmetrics
+## Configuring `checkmetrics`
 
 A metrics machine must have a valid `checkmetrics` configuration file in `/etc/checkmetrics`
 in order to verify the metrics CI results.
 
 To aid setup of this file on the bare metal metrics slaves, the `add_checkmetrics.yaml`
-ansible script can be used. A local checkmetrics toml file must be present in the same
+Ansible script can be used. A local `checkmetrics` TOML file must be present in the same`
```

Comment:
`ansible` ?

Reply:
yep, as in https://www.ansible.com/ ... not sure what your intention was @devimc - just surprised, or you wanted the capitalisation turned back? :-)

Reply:
Oh, I see - you mean `code block` it - I see....

Reply:
I was treating this as the "project name" (Ansible) but should it be treated as a command name then?

Reply:
I think in that context it is the project name.